biz.aQute.repository/bnd.bnd

```diff
@@ -26,6 +26,7 @@
 Export-Package: \
 	aQute.bnd.deployer.http;bnd-plugins=true,\
 	aQute.p2.api,\
+	aQute.p2.packed,\
```

Comment:
I would avoid exporting this package. There is no need I can see to make this API.